README.md

```diff
@@ -25,5 +38,6 @@ How do get started:
 5. Build and run ingester:
     ```
     cd ingester
+    make build
```

Comment:
This line is unnecessary because the `run` rule already builds the ingester binary